alphabet.py

```diff
@@ -0,0 +1,29 @@
+from graph import Graph
+
+
+class Alphabet:
+    def __init__(self):
+        self.letters = set()
+        self.relations = Graph()
+
+    def _compare_words(self, next_word, prev_word):
+        if len(prev_word) == 0:
+            return
+
+        for i in range(min(len(next_word), len(prev_word))):
+            p = prev_word[i]
+            n = next_word[i]
+
+            if p != n:
+                if n not in self.letters:
+                    self.letters.add(n)
```

Comment:
Is this add() the one defined in the file "graph.py"? Because in that case it would be missing an input in this call